tests/unittests/core/utils/test_working_dir.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Collection of tests for :mod:`orion.core.utils.working_dir`."""
+import os
+
+from orion.core.utils.working_dir import WorkingDir
+
+
```

Comment:
You should add a test for when folder exists and temp is True to make sure it does not crash.